chart/templates/preflight.yaml

```diff
@@ -0,0 +1,127 @@
+  name: {{ .Release.Name }}-preflight-config
+  namespace: {{ .Release.Namespace | quote }}
+stringData:
+  preflight.yaml: |
+    apiVersion: troubleshoot.sh/v1beta2
+    kind: Preflight
+    metadata:
+      name: preflight
+    spec:
+      collectors:
+        - postgres:
+            collectorName: workspace-db
+            uri: postgresql://{{ include "carto.postgresql.adminUser" . | trimAll "\"" }}:{{ .Values.externalPostgresql.adminPassword }}@{{ include "carto.postgresql.host" . | trimAll "\"" }}:{{ include "carto.postgresql.port" . | trimAll "\"" }}/{{ include "carto.postgresql.adminDatabase" . | trimAll "\"" }}
+        - registryImages:
+            images:
+              - {{ template "carto.accountsWww.image" . }}
+              - {{ template "carto.cdnInvalidatorSub.image" . }}
+              - {{ template "carto.httpCache.image" . }}
+              - {{ template "carto.importApi.image" . }}
+              - {{ template "carto.importWorker.image" . }}
+              - {{ template "carto.ldsApi.image" . }}
+              - {{ template "carto.mapsApi.image" . }}
+              - {{ template "carto.notifier.image" . }}
+              - {{ template "carto.router.image" . }}
+              - {{ template "carto.sqlWorker.image" . }}
+              - {{ template "carto.workspaceApi.image" . }}
+              - {{ template "carto.workspaceMigrations.image" . }}
+              - {{ template "carto.workspaceSubscriber.image" . }}
+              - {{ template "carto.workspaceWww.image" . }}
+      analyzers:
+        - registryImages:
+            checkName: Carto Registry Images
+            outcomes:
+              - fail:
+                  when: "missing > 0"
+                  message: Images are missing from registry
+              - warn:
+                  when: "errors > 0"
+                  message: Failed to check if images are present in registry
+              - pass:
+                  message: All Carto images are availables
+        - postgres:
+            checkName: PostgreSQL is available
+            collectorName: workspace-db
+            outcomes:
+              - fail:
+                  when: connected == false
+                  message: Cannot connect to PostgreSQL server
+              - pass:
+                  when: connected == true
+                  message: The PostgreSQL server is available
+        - postgres:
+            checkName: PostgreSQL must be v11.x or later
+            collectorName: workspace-db
+            outcomes:
+              - fail:
+                  when: connected == false
+                  message: Cannot connect to PostgreSQL server
+              - fail:
+                  when: version < 11.x
+                  message: The PostgreSQL server must be at least version 11
+              - pass:
+                  message: The PostgreSQL verion checks out
+        - clusterVersion:
+            outcomes:
+              - fail:
+                  when: "< 1.20.0"
```

Comment:
In our old documentation we had as minimum the `1.12` but looks too old. Whats your opinion here @moicalcob ?

(This is not a blocker for the PR but is something that we should decide in some way)

Reply:
I've been checking it and it seems that the `1.12` version is more than 5 years old. IMHO we should update it and ask at least for the `1.25`version 🤔

Reply:
Ok we can put min 1.25 and recommended >= 1.26

Reply:
We should leave the code as it is now but I should add to the AD the official upgrade of the version and the doc changes

Reply:
`1.25` looks a decent version by the time customers start using this. in GKE: https://cloud.google.com/kubernetes-engine/docs/release-schedule#schedule-for-release-channels